cartridges/app_custom_pdp/cartridge/controllers/Product.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+var server = require("server");
+server.extend(module.superModule);
+
+server.append("Show", function (req, res, next) {
+    var viewData = res.getViewData();
+
+    var ContentMgr = require("dw/content/ContentMgr");
+    var asset = ContentMgr.getContent("promotional-message");
```

Comment:
Is this asset called multiple times? If not we can just include it in the template with iscontentasset